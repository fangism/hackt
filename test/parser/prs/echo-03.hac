// "echo-03.hac"
// copy-modified from "echo-01.hac"
// testing the 'echo' PRS macro
// should die on reference to unknown node.  

defproc echo(bool p, q, r, s, z) {
prs {
	echo(r, s)
	echo(p, buh)		// death
	(p & q) | (r & s)	-> z-
	~((p & q) | (r & s))	-> z+
}
}

echo	E;
// E.p = E.r;

