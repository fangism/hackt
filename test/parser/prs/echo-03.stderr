object "buh" not found, ERROR!  [9:10..12]
ERROR: at least one error in PRS body.
